queries.sql

```diff
@@ -8,3 +8,46 @@ SELECT (name, escape_attempts) FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true ;
 SELECT * FROM animals WHERE name IS DISTINCT FROM 'Gabumon' ;
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3 ;
+
+begin;
+UPDATE animals
+SET species = 'unspecified';
+ROLLBACK;
+
+begin;
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS null;
+COMMIT;
+
+begin;
+DELETE FROM animals;
+rollback;
+
+begin;
+DELETE FROM animals
+WHERE date_of_birth > '2022-01-01';
+
+SAVEPOINT my_save;
+
+UPDATE animals
+SET weight_kg = weight_kg * (-1);
+
+rollback to my_save;
+
+UPDATE animals
+SET weight_kg = weight_kg * (-1)
+WHERE weight_kg < 0;
+
+commit;
```

Comment:
- [ ] OPTIONAL: It would be nice if after each commit, delete and rollback, you verify the data with a SELECT statement.  Otherwise, this is super awesome1 💪

Reply:
I understand that. It is exactly what I did in my terminal. But I didn't put it in the code. I guess it will make the code review easier as well. I will definitely take your suggestion into account. Thank you for your feedback.